### ISSUE TITLE
Cannot import dash-html-components with dash==0.23.1 and python>=3.7

### ISSUE THREAD

**Issue by rmarren1** [plotly/dash#327]
See discussion here: https://community.plot.ly/t/error-running-dash-after-upgrade-of-plotly/12507/5?u=chriddyp

A `SyntaxError: invalid syntax` is thrown at dash_html_components/Script.py, line 48 when attempting to import `dash-html-components` with `dash==0.23.1` and `python>=3.7`.

Fix is to downgrade dash to 0.22.0, patch will be merged soon: https://github.com/plotly/dash/pull/325

**Comment by rmbarley** [comment]
This is still not working for me with dash==0.26.6. I tried reinstalling and downgrading to dash==0.24.1, and it still fails with the same syntax error.

**Comment by rmarren1** [comment]
Try `dash-html-components==0.12.0`. It looks like `0.13.0` was built against py2.7 or py3.6 causing this error.

**Comment by T4rk1n** [comment]
@rmarren1 I just released 0.13.0 with the dev bundle support, I didn't touch the .py files and didn't regenerate them. The .py files in the repo are bad.

**Comment by rmarren1** [comment]
Yeah the files in the repo are against 3.6. I'll change them to be against 3.7 and release `0.13.1` to fix this

**Comment by rmarren1** [comment]
@rmbarley Should work with `dash-html-components==0.13.2` just released as a patch
